content/blog/2018/12/2018-12-14-java11-preview-availability.adoc

```diff
@@ -0,0 +1,237 @@
+
+In order to simplify testing, we also provide a
+experimental images on DockerHub.
+We have also set up a continuous delivery flow for them,
+so you can get patches without waiting for Jenkins weekly releases.
+
+//TODO: Host on jenkinsci and replace by "blueocean"
+
+* link:https://hub.docker.com/r/jenkins4eval/blueocean-platform-support/[jenkins4eval/blueocean-platform-support] -
+Equivalent of link:https://hub.docker.com/r/jenkinsci/blueocean/[jenkinsci/blueocean]
+** Java 11 tag: `latest-jdk11`
+** The image bundles all Jenkins Pipeline and Blue Ocean patches required to run on Java 11
+** If you want to try Pipeline, use this image
+* link:https://hub.docker.com/r/jenkins/jenkins-experimental/[jenkins/jenkins-experimental] -
+Equivalent of link:https://hub.docker.com/r/jenkins/jenkins/[jenkins/jenkins]
+** The image is released from the `java11-support` feature branch in the Jenkins core
+** The branch may be slightly ahead or behind the `master` branch,
+   we may use the branch to quickly deliver patches to Java 11 users
+
+Eventually we will move the experimental flow to the new `jenkins4eval` organization
+being created as a part of jep:TODO[].
+
+=== Running jenkins.war with Java 11
+
+Running without Docker is not that trivial,
+because Jenkins depends on some modules which have been removed from Java 11.
+We plan to address it in the General Availability release somehow (see jira:JENKINS-52186[]),
+but for now some manual actions are required to run Jenkins WAR with Java 11.
+
+1. Download Jenkins WAR for 2.155
+2. Download the following libraries to the same directory as jenkins.war
+** link:http://central.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar[jaxb-api-2.3.0.jar] (save as jaxb-api.jar)
+** link:http://central.maven.org/maven2/com/sun/xml/bind/jaxb-core/2.3.0.1/jaxb-core-2.3.0.1.jar[jaxb-core-2.3.0.1.jar] (save as jaxb-core.jar)
+** link:http://central.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.0.1/jaxb-impl-2.3.0.1.jar[jaxb-impl-2.3.0.1.jar] (save as jaxb-impl.jar)
+** https://github.com/javaee/activation/releases/download/JAF-1_2_0/javax.activation.jar[javax.activation v.1.2.0]  (save as javax.activation.jar)
+3. Run the following command:
+
+```shell
+Run Jenkins with ${JAVA11_HOME}/bin/java \
+    -p jaxb-api.jar:javax.activation.jar --add-modules java.xml.bind,java.activation \
+    -cp jaxb-core.jar:jaxb-impl.jar \
+    -jar jenkins.war --enable-future-java --httpPort=8080 --prefix=/jenkins
+```
+
+=== Java 11 compatibility issues
+
+To help users to track down the compatibility issues,
+we have created a new link:https://wiki.jenkins.io/display/JENKINS/Known+Java+11+Compatibility+issues[Known Java 11 Compatibility Issues] Wiki page.
+
+One of the major blockers we hit was the release of the plugin:workflow-support[Pipeline: Support Plugin]
+with jira:JENKINS-51998[] fixes,
+but we worked it around by deploying an experimental fix to the
```

Comment:
"worked it around" looks unnatural to me, "worked around it" would sound better to me.
Maybe a native speaker could chime in? Cc @jeffpearce maybe?